website/src/main/java/com/google/step/servlets/DataServlet.java

```diff
@@ -35,10 +39,13 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    String filename = "../../../similarity/sample_data.csv";
```

Comment:
Please take a look at https://github.com/googleinterns/step3-2020/pull/14/commits/a0f87577dfd08ad648b90002b4468f7c504c1386#diff-987d8db52973867baf7d333fa93a1420 as that is how you could safely handle reading CSVs.
In the future, as Ali pointed out, it may be more beneficial for us to move the csv to a database like Datastore to serve